src/data/README.md

```diff
@@ -0,0 +1,68 @@
+# Sampling Dolma
+
+As of 5/28/24, the [Dolma dataset sample](https://huggingface.co/datasets/orionweller/dolma_20_percent_sample) has ~330bn tokens (see `source_stats.csv`).  This can be re-created by runnning:
+
+```
+python sample_from_dolma.py --repo_name orionweller/dolma_20_percent_sample --percentage 0.2
+```
+
+# Generating Mixtures
+
+Then, to create different mixtures from this data, run
+
+```
+python dolma_mixtures.py --dolma_hf_path {{repo_name from above}} --config_fn {{.yaml file in --config_dir}}
+```
+
+## Config Docs
+
+The config yaml file specifies how much of each dataset source to use, in addition to specifying an overall desired token count of the mixture. It follows the following structure:
+
+```
+sources:
+  - name: cc_en_middle
+    source_coefficient: 1
+  - name: falcon
+    source_coefficient: 1
+  - name: cc_en_tail
+    source_coefficient: 1
+  - name: cc_en_head
+    source_coefficient: 1
+  - name: c4
```

Comment:
We should keep it as-is for now, but following Enrico's comment, we should keep in mind this is low quality as we do the ablations (and we've got options if needed, e.g. fineweb or just using higher quality subsets)